deeploy.sh

```diff
@@ -0,0 +1,8 @@
+export PULP_SDK_HOME=${DEEPLOY_INSTALL_DIR}/pulp-sdk
+export LLVM_INSTALL_DIR=${DEEPLOY_INSTALL_DIR}/llvm
+export PULP_RISCV_GCC_TOOLCHAIN=
+export MEMPOOL_HOME=${DEEPLOY_INSTALL_DIR}/mempool
```

Comment:
You shouldn't just set all these paths relative to `DEEPLOY_INSTALL_DIR`. The Makefile is set up such that previous installations may be reused, and each tool gets its own `DIR` variable.